testkit/core/src/main/scala/com/lightbend/lagom/internal/testkit/InternalSubscriberStub.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2009-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.testkit
+
+import akka.Done
+import akka.actor.ActorRef
+import akka.stream.{ Materializer, OverflowStrategy }
+import akka.stream.scaladsl.{ Flow, Keep, Sink, Source }
+
+import scala.concurrent.Future
+
+private[lagom] class InternalSubscriberStub[Message](
```

Comment:
Looks good! Maybe they should be declared `protected` to make it clear that they're only intended to be used from inside the language-specific bindings?